tests/scripts/thread-cert/node.py

```diff
@@ -338,6 +370,10 @@ def get_addr64(self):
         self._expect('Done')
         return addr64
 
+    def set_addr64(self, addr64):
```

Comment:
nit: add [typing](https://docs.python.org/3/library/typing.html) for both `set_addr64` and `get_addr64` because people might think that `addr64` is of type `uint64`.

Reply:
@bukepo